res/style.css

```diff
@@ -0,0 +1,73 @@
+html {
+    background-image: url("/img/markus-winkler-IrRbSND5EUc-unsplash.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
+    overflow: hidden;
+}
+
+body {
+    display: grid;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    overflow: hidden;
+    color: white;
+    font-family: 'Montserrat';
+}
+
+#heading {
+    font-size: 5em;
+    margin: 0 0;
+
+}
+
+main {
+    text-shadow: rgba(0, 0, 0, 0.5) 0px 0px 15px;
+    background-color: rgba(0, 0, 0, 0.3);
+    padding: 5em;
+    backdrop-filter: blur(10px) grayscale();
```

Comment:
The `backdrop-filter` property is not supported by firefox (at least per default, it can still be enabled). But I think this isn't a big problem though